code/game/gamemodes/miniantags/guardian/types/healer.dm

```diff
@@ -52,8 +52,8 @@
 			changeNext_move(CLICK_CD_MELEE)
 			if(heal_cooldown <= world.time && !stat)
 				var/mob/living/carbon/C = target
-				C.adjustBruteLoss(-5, robotic=1)
```

Comment:
This will break healing of IPCs and robotic limbs. You can either cast target into a humanoid (adjustBrute/Burn has robotic as a param there) or add it to the params of the base proc.

Reply:
IPCs still heal
![image](https://user-images.githubusercontent.com/25063394/77247889-2f3bd580-6c2d-11ea-852d-a0022bcaf3ae.png)

Reply:
Edit: Will fix guardians now